_apidocgen/contract.py

```diff
@@ -0,0 +1,238 @@
+"""Contract parser.
+
+Loads and parses contract artefacts and converts data into Markdown.
+"""
+
+import logging
+import re
+from enum import Enum
+from os import path
+from typing import TypeAlias
+
+from . import DEBUG
+from .paths import Paths
+from .markdown import MarkdownDocument
+
+NATSPEC_VERSION = 1
```

Comment:
I can't see the `kind` field?

https://docs.soliditylang.org/en/latest/natspec-format.html#documentation-output

>Note
>
>Starting Solidity version 0.6.11 the NatSpec output also contains a version and a kind field. Currently the version is set to 1 and kind must be one of user or dev. In the future it is possible that new versions will be introduced, deprecating older ones.

Reply:
I assumed that the file extensions are correct, i.e. if it's `*.docuser` then `kind` is `user` and if it's `*.docdev` then `kind` is `dev`. Nevertheless I could add an assertion that checks it.